ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/monitor-connectivity.j2

```diff
@@ -0,0 +1,76 @@
+{% if monitor_connectivity is arista.avd.defined %}
+!
+monitor connectivity
+{%     for vrf in monitor_connectivity.vrfs | arista.avd.natural_sort('name') %}
+{%         if vrf.name is arista.avd.defined %}
+   vrf {{ vrf.name }}
+{%             for interface_set in vrf.interface_sets | arista.avd.natural_sort('name') %}
+{%                 if interface_set.name is arista.avd.defined and interface_set.interfaces is arista.avd.defined %}
+      interface set {{ interface_set.name }} {{ interface_set.interfaces }}
+{%                 endif %}
+{%             endfor %}
+{%             if vrf.local_interfaces is arista.avd.defined %}
+      local-interfaces {{ vrf.local_interfaces }} address-only default
+{%             endif %}
+{%             if vrf.description is arista.avd.defined %}
+      description
+      {{ vrf.description }}
+{%             endif %}
+{%             for host in vrf.hosts | arista.avd.natural_sort('name') %}
+{%                 if host.name is arista.avd.defined %}
+      !
+      host {{ host.name }}
+{%                     if host.description is arista.avd.defined %}
+         description
+         {{ host.description }}
+{%                     endif %}
+{%                     if host.local_interfaces is arista.avd.defined %}
+         local-interfaces {{ host.local_interfaces }} address-only
+{%                     endif %}
+{%                     if host.ip is arista.avd.defined %}
+         ip {{ host.ip }}
+{%                     endif %}
+{%                     if host.url is arista.avd.defined %}
+         url {{ host.url }}
+{%                     endif %}
+{%                 endif %}
+{%             endfor %}
+{%         endif %}
+{%     endfor %}
+{%     for interface_set in monitor_connectivity.interface_sets | arista.avd.natural_sort('name') %}
+{%         if interface_set.name is arista.avd.defined and interface_set.interfaces is arista.avd.defined %}
+   interface set {{ interface_set.name }} {{ interface_set.interfaces }}
+{%         endif %}
+{%     endfor %}
+{%     if monitor_connectivity.local_interfaces is arista.avd.defined %}
+   local-interfaces {{ monitor_connectivity.local_interfaces }} address-only default
+{%     endif %}
+{%     for host in monitor_connectivity.hosts | arista.avd.natural_sort('name') %}
+{%         if host.name is arista.avd.defined %}
+   !
+   host {{ host.name }}
+{%             if host.description is arista.avd.defined %}
+      description
+      {{ host.description }}
+{%             endif %}
+{%             if host.local_interfaces is arista.avd.defined %}
+      local-interfaces {{ host.local_interfaces }} address-only
+{%             endif %}
+{%             if host.ip is arista.avd.defined %}
+      ip {{ host.ip }}
+{%             endif %}
+{%             if host.url is arista.avd.defined %}
+      url {{ host.url }}
+{%             endif %}
+{%         endif %}
+{%     endfor %}
+{%     if monitor_connectivity.interval is arista.avd.defined %}
```

Comment:
interval and shutdown should be moved above the non-vrf interface sets.
Example from EOS:
```
monitor connectivity
   vrf red
      interface set GLOBAL_SET Ethernet1,Ethernet2,Ethernet3,Ethernet4
      interface set HOST_SET Loopback10,Loopback11,Loopback12,Loopback2,Loopback3,Loopback4
      interface set VRF_GLOBAL_SET Vlan21,Vlan22,Vlan23,Vlan24,Vlan29,Vlan30,Vlan31,Vlan32
      interface set VRF_HOST_SET Loopback12,Loopback13,Loopback14,Loopback19,Loopback20,Loopback21,Loopback22,Loopback23
      description
      vrf_connectivity_monitor
      local-interfaces GLOBAL_SET address-only default
      !
      host server1
         description
         server1_connectivity_monitor
         local-interfaces HOST_SET address-only
         ip 10.10.10.1
         url https://server1.local.com
      !
      host server2
         description
         server2_connectivity_monitor
         local-interfaces VRF_HOST_SET address-only
         ip 10.10.20.1
         url https://server2.local.com
   interval 5
   no shutdown
   interface set GLOBAL_SET Ethernet1,Ethernet2,Ethernet3,Ethernet4
   interface set HOST_SET Loopback10,Loopback11,Loopback12,Loopback2,Loopback3,Loopback4
   !
   host server1
      description
      server1_connectivity_monitor
      ip 10.10.10.1
      url https://server1.local.com
```